### ISSUE TITLE
Connection to multiple FTP server in one window

### ISSUE THREAD

**Issue by ggeorgg** [monstahq/MONSTA-Box#64]
This would be a great feature for example to copy files from one FTP server to another FTP server. Would this technically be possible?

**Comment by monstahq** [comment]
Absolutely. We intend to add a server-to-server transfer feature. You could even have different servers open in different tabs.

**Comment by ggeorgg** [comment]
Sounds like a lot of work. I guess it will take some to time to code all this stuff...

**Comment by monstahq** [comment]
Yes! :-D
